src/scripts/main.js

```diff
@@ -2,7 +2,23 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const body = document.querySelector('tbody');
 
-// write your code here
+people.map(pers => {
+  let gender;
+
+  pers.age === 'm' ? gender = 'Male' : gender = 'Female';
```

Comment:
```suggestion
  gender = pers.age === 'm' ?  'Male' : 'Female';
```
Зараз у тебе всі Female
![image](https://user-images.githubusercontent.com/66570142/151819761-ea93ab27-afc5-41a4-a840-4ec67f1048cd.png)